server/api/api.go

```diff
@@ -195,15 +194,16 @@ func (a *API) errorResponse(w http.ResponseWriter, r *http.Request, err error) {
 	case model.IsErrNotImplemented(err):
 		errorResponse.ErrorCode = http.StatusNotImplemented
 	default:
-		a.logger.Error("API ERROR",
-			mlog.Int("code", http.StatusInternalServerError),
-			mlog.Err(err),
-			mlog.String("api", r.URL.Path),
-		)
 		errorResponse.Error = "internal server error"
 		errorResponse.ErrorCode = http.StatusInternalServerError
 	}
 
+	a.logger.Warn("api error response",
+		mlog.Int("code", http.StatusInternalServerError),
```

Comment:
Should this be the error code that was set in the cases above instead of the default for all?

```suggestion
		mlog.Int("code", errorResponse.ErrorCode),
```

Reply:
Ah, goodness, yes! Sorry for that obvious slip up :/